2016-contributing-to-your-career.md

```diff
@@ -0,0 +1,189 @@
+
+…and…
+
+> “Open source has expanded my network in a way I could not have done on my own.”
+
+You hear people like Matthew Garret, a well-known Linux contributor, [saying](http://mjg59.dreamwidth.org/30420.html)
+
+> “Without free software I'd have started years later. I'd have lost the opportunity to collaborate with people spread over the entire world. My first job would have looked very different, as would my entire career since then. Without free software, almost everything I've achieved in my adult life would have been impossible.”
+
+Participating in open source will set you on path of continuously learning, building amazing things with amazing people, and finding great opportunities.
+
+Contents:
+
+1. [The value of contributing](#the-value-of-contributing)
+2. [Becoming a contributor](#becoming-a-contributor)
+3. [Building a reputation](#building-a-reputation)
+
+## The value of contributing
+
+Participating in open source has incredible advantages, like…
+
+- …working with and learning from some of the best developers in the world…
+- …on interesting technology that gets used by hundreds, thousands, or even millions of people…
+- …from wherever in the world you call "home"…
+- …with free tools that you love or want to learn to love…
+- …in communities with people that share your passions…
+- …all for free.
+
+Those alone are great reasons.  This talk takes those for granted, and instead focuses on something more selfish: your career.
+
+**What do Disney, Walmart, and John Deer have in common?** One is an "American diversified multinational mass media and entertainment conglomerate" ([wikiepdia](https://en.wikipedia.org/wiki/The_Walt_Disney_Company), another is the largest retailer in the world, and the third makes tractors.
+
+They're all software companies, and they all consume and contribute to open source software. They do other things too, but they are becoming primarily software companies. Them any every other company in the world, including:
+
+> AirBnB. Amazon. AMD. Apple. Autodesk. Bloomberg. Bosch. Box. Broadcom. Capital One. Cisco. Citrix. Comcast. Craigslist. DellEMC. Disney. Dropbox. Ebay. Facebook. Ford. Fujitsu. GE. GitHub. Goldman Sachs. Google. Guardian. Home Depot. Honda. HP. IBM. Intel. J.P. Morgan. John Deere. Lenovo. LG. Mazda. Microsoft. NEC. Netflix. Nike. Nissan. Nokia. Nvidia. Oracle. Panasonic. PayPal. Pinterest. Qualcomm. Rackspace. RedHat. Salesforce. Samsung. SAP. Semantec. Sony. Square. Stripe. TMobile. Target. Toshiba. Toyota. Twitter. VMWare. Walmart. Western Digital. Yahoo.
+
+Every company is becoming a software company, the future of their business depends on open source software. Which means, the future of their business relies on the people that contribute to open source software. They are all quickly realizing this. So, they want to hire people that have demonstrated experience with those technologies that can continue to evolve them.
```

Comment:
"that can continue to evolve them." &rarr; what "them" is referring to is unclear. Does it refer to the technologies? The companies? Or the people?